### DISCUSSION TITLE
Works great with a Amcrest IP8M-T2599EW

### DISCUSSION THREAD

**Discussion by kbmetz** [mmcc-xx/BirdCAGE#6] [General]
Just wanted to say I was able to get this working with an Amcrest IP8M-T2599EW. I've already got the camera setup and running with Frigate, and I was able to add this docker container and I've got over 30 detections and they look valid considering the area I live and the sounds I hear. Very cool!

**Comment by mmcc-xx** [comment]
That's great to hear!
